apps/docs/src/content/learn/basics/disposing-objects.mdx

```diff
@@ -42,7 +42,7 @@ components. This disables disposal for the entire subtree.
 ```svelte
 <script>
   import { T, useTexture } from '@threlte/core'
-  import { BoxBufferGeometry, MeshStandardMaterial } from 'three'
+  import { BoxGeometry, MeshStandardMaterial } from 'three'
```

Comment:
Looks like we can remove this altogether? (it's just used in a T component)

Reply:
yeah we could replace with T.BoxGeometry in each case. same for the material too.

Reply:
I mean in this case this import is completely unused

Reply:
oh i see okay. i'll get that outta there.